src/ontology/Makefile

```diff
@@ -75,6 +75,10 @@ all_odk: odkversion test all_assets
 test: odkversion reason_test sparql_test robot_reports $(REPORTDIR)/validate_profile_owl2dl_$(ONT).owl.txt
 	echo "Finished running all tests successfully."
 
+.PHONY: test
+test_fast:
```

Comment:
@matentzn I thought that `update_repo` only made updates based on `mondo-ingest-odk.yaml` based on what you said the other day, but I think what I should have taken away is that I will almost never need to use it unless I'm making changes to that file.

There are a few other changes not related to my `mondo-ingest-odk.yaml` updates that you can see in the diff, if you don't mind taking a look.

Reply:
Yes, this is because you get updates that are only available in the ODK `:dev` tag. Its fine!